cime_config/namelist_definition_drv.xml

```diff
@@ -2055,25 +2055,25 @@
       <value samegrid_atm_lnd='true'>idmap</value>
     </values>
   </entry>
-  <entry id="lnd2rof_map">
+  <entry id="lnd2rof_map" modify_via_xml="LND2ROF_FMAPNAME">
     <type>char</type>
     <category>mapping</category>
     <input_pathname>abs</input_pathname>
     <group>MED_attributes</group>
-    <desc>lnd to rof mapping, 'unset' or 'idmap' are normal possible values</desc>
+    <desc>lnd to rof mapping, 'unset' or 'idmap' are normal possible values (mapping file given for mizuRoute grids)</desc>
     <values>
-      <value>unset</value>
+      <value>$LND2ROF_FMAPNAME</value>
       <value samegrid_lnd_rof='true'>idmap</value>
```

Comment:
As this is written, samegrid_lnd_rof will override the LND2ROF_FMPANAME value. This is probably correct, but means there is hidden behavior where the LND2ROF_FMAPNAME value that the user sets is ignored.

Reply:
The one time this samegrid_lnd_rof comes up is for the nldas2_rnldas2_mnldas2 grid and I made sure with the new code it gives idmap as it should for that case.